composer.json

```diff
@@ -12,20 +12,26 @@
             "homepage": "https://sonata-project.org/"
         }
     ],
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/Ener-Getick/PHP-CS-Fixer"
+        }
+    ],
```

Comment:
Must be removed once https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1510 is merged.